cmake/modules/AddSwift.cmake

```diff
@@ -1606,6 +1606,98 @@ function(add_swift_library name)
+             ${SWIFTLIB_SWIFT_MODULE_DEPENDS_WATCHOS})
+      elseif(${sdk} STREQUAL FREEBSD)
+        list(APPEND swiftlib_module_depends_flattened
+             ${SWIFTLIB_SWIFT_MODULE_DEPENDS_FREEBSD})
+      elseif(${sdk} STREQUAL LINUX OR ${sdk} STREQUAL ANDROID)
+        list(APPEND swiftlib_module_depends_flattened
+             ${SWIFTLIB_SWIFT_MODULE_DEPENDS_LINUX})
+      elseif(${sdk} STREQUAL CYGWIN)
+        list(APPEND swiftlib_module_depends_flattened
+             ${SWIFTLIB_SWIFT_MODULE_DEPENDS_CYGWIN})
+      elseif(${sdk} STREQUAL HAIKU)
+        list(APPEND swiftlib_module_depends_flattened
+             ${SWIFTLIB_SWIFT_MODULE_DEPENDS_HAIKU})
+      endif()
+
+      # Collect architecture agnostic SDK framework dependencies
+      set(swiftlib_framework_depends_flattened ${SWIFTLIB_FRAMEWORK_DEPENDS})
+      if(${sdk} STREQUAL OSX)
+        list(APPEND swiftlib_framework_depends_flattened
+             ${SWIFTLIB_FRAMEWORK_DEPENDS_OSX})
+      elseif(${sdk} STREQUAL IOS OR ${sdk} STREQUAL IOS_SIMULATOR OR
+             ${sdk} STREQUAL TVOS OR ${sdk} STREQUAL TVOS_SIMULATOR)
+        list(APPEND swiftlib_framework_depends_flattened
+             ${SWIFTLIB_FRAMEWORK_DEPENDS_IOS_TVOS})
+      endif()
+
+      # Collect architecutre agnostic compiler flags
+      set(swiftlib_swift_compile_flags_all ${SWIFTLIB_SWIFT_COMPILE_FLAGS})
+      if(${sdk} STREQUAL OSX)
+        list(APPEND swiftlib_swift_compile_flags_all
+             ${SWIFTLIB_SWIFT_COMPILE_FLAGS_OSX})
+      elseif(${sdk} STREQUAL IOS OR ${sdk} STREQUAL IOS_SIMULATOR)
+        list(APPEND swiftlib_swift_compile_flags_all
+             ${SWIFTLIB_SWIFT_COMPILE_FLAGS_IOS})
+      elseif(${sdk} STREQUAL TVOS OR ${sdk} STREQUAL TVOS_SIMULATOR)
+        list(APPEND swiftlib_swift_compile_flags_all
+             ${SWIFTLIB_SWIFT_COMPILE_FLAGS_TVOS})
+      elseif(${sdk} STREQUAL WATCHOS OR ${sdk} STREQUAL WATCHOS_SIMULATOR)
+        list(APPEND swiftlib_swift_compile_flags_all
+             ${SWIFTLIB_SWIFT_COMPILE_FLAGS_WATCHOS})
+      elseif(${sdk} STREQUAL WINDOWS)
+        # FIXME(SR2005) static and shared are not mutually exclusive; however
+        # since we do a single build of the sources, this doesn't work for
+        # building both simultaneously.  Effectively, only shared builds are
+        # supported on windows currently.
+        if(SWIFTLIB_SHARED)
+          list(APPEND swiftlib_swift_compile_flags_all -D_USRDLL)
+          if(SWIFTLIB_IS_STDLIB_CORE)
+            list(APPEND swiftlib_swift_compile_flags_all -DswiftCore_EXPORTS)
+          endif()
+        elseif(SWIFTLIB_STATIC)
+          list(APPEND swiftlib_swift_compile_flags_all -D_LIB)
+        endif()
+      endif()
+
+
+      # Collect architecture agnostic SDK linker flags
+      set(swiftlib_link_flags_all ${SWIFTLIB_LINK_FLAGS})
+      if(${sdk} STREQUAL IOS_SIMULATOR)
+        if(${name} STREQUAL swiftMediaPlayer)
+          # message("DISABLING AUTOLINK FOR swiftMediaPlayer")
```

Comment:
I am not sure if you should comment this out. I think this caused problems in the past and the logging was meant to provide info if this happened.

@zisko @Rostepher is this useful for you guys? If not, we should just delete this line.